spec/jobs/post_update_topic_tracking_state_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Jobs::PostUpdateTopicTrackingState do
+  fab!(:post) { Fabricate(:post) }
+
+  it 'should publish messages' do
+    messages = MessageBus.track_publish { subject.execute({ post_id: post.id }) }
+    expect(messages.size).not_to eq(0)
```

Comment:
I feel like this assertion should be more specific to assert for the specific messages we're expecting. I think we need to assert for the right messages instead of just any messages.

Reply:
still this is pretty safe, it is saying that absolutely no messages are being published ... I think it is ok.

Reply:
Yeah we can certainly improve, but I would like to get this fix in ... seems pretty safe, tests should not flake.